docs/user-docs/latest/users-roles-permissions/configuring-end-users-roles.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Configure End-users Roles - Strapi User Guide
+description: Instructions to configure the end-users roles for a front-end application using the Users & Permissions plugin
+---
+
+# Configuring end-users roles
+
+End-users are the users who consume the content that is created and managed with a Strapi application and displayed on front-end applications (e.g. websites, mobile applications, connected devices etc.). Unlike the administrators, they do not have access to the admin panel. End-users are managed with the Users & Permissions plugin (see [Users & Permissions plugin](../plugins/strapi-plugins.md#users-permissions-plugin)). This plugin is however not entirely managed and configured from one same place of the admin panel: end-users accounts are managed from the Content Manager (see [Managing end-user accounts](../users-roles-permissions/managing-end-users.md)) but end-users roles and permissions are managed in the Settings interface.
+
+The configurations of the end-users roles and permissions are available in the *Users & Permissions plugin* section of the Settings interface, accessible from *General > Settings* in the main navigation of the admin panel.
+
+![End-users roles interface](../assets/users-permissions/end-user_roles.png)
+
+The *Roles* sub-section of *Users & Permissions plugin* displays all created roles for the end-users of your Strapi application.
+
+From this interface, it is possible to:
+
+- create a new end-user role (see [Creating a new role](#creating-a-new-role)),
+- delete an end-user role (see [Deleting a role](#deleting-a-role)),
+- or access information regarding an end-user role, and edit it (see [Editing a role](#editing-a-role)).
+
+By default, 2 end-user roles are defined for any Strapi application:
+
+- Authenticated: for end-users to access content only if they are logged in to a front-end application.
+- Public: for end-users to access content without being logged in to a front-end application.
+
+::: tip NOTE
+The end-user role attributed by default to all new end-users can be defined in the *Advanced settings* sub-section of *Users & Permissions plugin* (see [Configuring advanced settings](../settings/configuring-users-permissions-plugin-settings.md#configuring-advanced-settings)).
+:::
+
+## Creating a new role
+
+On the top right side of the *Users & Permissions plugin > Roles* interface, an **Add new role** button is displayed. It allows to create a new role for end-users of your Strapi application.
```

Comment:
Just wondering if we consider the `+` sign as part of the button name or not 🤔 (**+ Add new role**).

Just asking for consistency, because I used it a lot in the Quick Start Guide and was always including the plus sign.

Reply:
Personally I don't. I checked the whole user guide just to be sure and I haven't written the `+` anywhere so I guess for consistency we can continue to skip it 🤔

Reply:
Sounds good, thank you Mégane!